lib/sinon/match.js

```diff
@@ -178,6 +178,18 @@ match.same = function (expectation) {
     }, "same(" + valueToString(expectation) + ")");
 };
 
+match.in = function (arrayOfExpectations) {
+    if (!(arrayOfExpectations instanceof Array)) {
```

Comment:
Please use [`Array.isArray`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray) to accurately detect instances of Array.

See http://web.mit.edu/jwalden/www/isArray.html for a detailed explanation.